MongoDB/0.1.0/docker-compose.yml

```diff
@@ -16,11 +15,10 @@ mongo-cluster:
     - mongo-base
 mongo-base:
   restart: always
-  tty: true
   labels:
     io.rancher.container.hostname_override: container_name
     io.rancher.container.start_once: true
-  image: rancher/mongodb-conf:v0.1.0
+  image: husseingalal/mongodb-conf:v0.2.0
```

Comment:
@cloudnautique yeah that's wrong, I was testing with my image and forgot to change it